Jenkinsfile

```diff
@@ -117,29 +118,23 @@ stage('Configure') {
 			jdk: [
 					// This should not include every JDK; in particular let's not care too much about EOL'd JDKs like version 9
 					// See http://www.oracle.com/technetwork/java/javase/eol-135779.html
-					new JdkBuildEnvironment(version: '17', buildJdkTool: 'OpenJDK 17 Latest',
+					new JdkBuildEnvironment(testJavaVersion: '17', testCompilerTool: 'OpenJDK 17 Latest',
 							condition: TestCondition.BEFORE_MERGE,
 							isDefault: true),
-					new JdkBuildEnvironment(version: '8', buildJdkTool: 'OracleJDK8 Latest',
+					new JdkBuildEnvironment(testJavaVersion: '8', testLauncherTool: 'OracleJDK8 Latest',
```

Comment:
Before you ask... no, I don't know why we only override the launcher, and not the compiler, when using JDK8. I just know that's how it's done in Hibernate Search, so I didn't take any risk and did the same here.

Reply:
I cleared that up: using a JDK8 compiler with `--release` (which is what we use in pom.xml now) leads to fatal errors. So, it's just simpler to compile with JDK11 to Java 8 bytecode.